codalab/worker/main.py

```diff
@@ -1,8 +1,3 @@
-#!/usr/bin/env python3.6
```

Comment:
Why remove this line?

Reply:
We never should execute this script directly